admin/react/assets/src/dashboard-app/pages/welcome/Welcome.js

```diff
@@ -0,0 +1,118 @@
+import CustomFontGrid from "./CustomFontGrid";
+import CustomFontList from "./CustomFontList";
+import SearchBar from "./SearchBar";
+import { Link } from "react-router-dom";
+import { __ } from "@wordpress/i18n";
+
+const Welcome = () => {
+	const [activeView, setActiveView] = useState("list");
+	const [searchResults, setSearchResults] = useState(null);
+	const [loading, setLoading] = useState(false);
+	const toggleView = (value) => {
+		setActiveView(value);
+	};
+	return (
+		<div className="">
+			<div className="bg-white border-b border-slate-200">
+				<div className="max-w-3xl mx-auto px-3 sm:px-6 lg:max-w-full">
+					<div className="relative py-7">
+						<div className="flex flex-col lg:flex-row justify-between items-start">
+							<div>
+								<h2 className="text-[1.625rem] font-semibold mb-3.5">
+									{ __( "Custom Fonts", "custom-fonts" ) }
+								</h2>
+								<p className="text-xs">
+									{ __( "Manage fonts that are used on your website.", "custom-fonts" ) }
+								</p>
+							</div>
+							<div className="relative tablet:mt-4">
+								<Link
+									to={{
+										pathname: "admin.php",
+										search: `?page=bsf-custom-fonts&path=add-fonts`,
+									}}
+									className="flex text-sm text-white hover:text-white bg-primary px-3 py-2"
+								>
+									{ __("Add New Font", "custom-fonts") }
+								</Link>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+			<div className="max-w-3xl mx-auto px-3 sm:px-6 lg:max-w-full">
+				{/* Search Custom Font */}
+				<SearchBar setSearchResults={ setSearchResults } setLoading={ setLoading }/>
+				{/* Font Counter and View toggle */}
+				<div className="flex justify-between items-center my-6">
+					<div className="text-base"> { searchResults ? searchResults.found_posts : bsf_custom_fonts_admin.fontPostCount } { __(" font families", "custom-fonts") } </div>
+					<div className="flex justify-end gap-x-7">
+						<div
+							className="cursor-pointer"
+							onClick={() => toggleView("list")}
+						>
+							<svg
+								width="20"
+								height="20"
+								viewBox="0 0 20 20"
+								fill="none"
+								xmlns="http://www.w3.org/2000/svg"
+							>
+								<path
+									fillRule="evenodd"
+									clipRule="evenodd"
+									d="M2 4.75C2 4.33579 2.33579 4 2.75 4H17.25C17.6642 4 18 4.33579 18 4.75C18 5.16421 17.6642 5.5 17.25 5.5H2.75C2.33579 5.5 2 5.16421 2 4.75ZM2 10C2 9.58579 2.33579 9.25 2.75 9.25H17.25C17.6642 9.25 18 9.58579 18 10C18 10.4142 17.6642 10.75 17.25 10.75H2.75C2.33579 10.75 2 10.4142 2 10ZM2 15.25C2 14.8358 2.33579 14.5 2.75 14.5H17.25C17.6642 14.5 18 14.8358 18 15.25C18 15.6642 17.6642 16 17.25 16H2.75C2.33579 16 2 15.6642 2 15.25Z"
+									fill={
+										activeView === "list"
+											? "#1D2327"
+											: "#DDDDDD"
+									}
+								/>
+							</svg>
+						</div>
+						<div
+							className="cursor-pointer"
+							onClick={() => toggleView("grid")}
+						>
+							<svg
```

Comment:
Same here @imnavanath regarding svg reuse

Reply:
Added separate task in backlog for this - https://brainstormforce.atlassian.net/browse/CFP-38